### DISCUSSION TITLE
Consider Remove or Refactor `Getting_started_with_the_web` Section in Learn Area

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [mdn/mdn-community#658] [Content]
I found that, for example, https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web/HTML_basics is similar to https://developer.mozilla.org/docs/Learn/HTML/Introduction_to_HTML/Getting_started. In fact, I notice that in `getting-started` we have more friendly content to beginners like `Installing_basic_software`. But I think we shouldn't serve duplicate content here.

**Comment by dipikabh** [comment]
Hi @awxiaoxian2020, we appreciate your feedback.

I've opened an issue to track this work: https://github.com/mdn/content/issues/33046.
